managed-pointer/cpp-20/D0567R1.md

```diff
@@ -229,94 +391,105 @@ auto gpuExecutor = gpuContext.executor();
 auto gpuAllocator = gpuContext.allocator();
 
 /* Construct a managed_ptr ptraA allocated on the host CPU */
-std::experimental::execution::managed_ptr<float> ptrA(1024);
+execution::managed_ptr<my_data> ptrA();
```

Comment:
There is a requirement that the `managed_ptr` does not call the constructor of the object it is managing, this is specified earlier in the paper. The reason for this is that the `managed_ptr` is either constructed from a pointer in which case the object has already been constructed or it is constructed from an allocator in which case it is not always possible for an executor's allocator to call the constructor on the alllocated memory.